verl/models/qwen2/megatron/checkpoint_utils/qwen2_loader.py

```diff
@@ -66,11 +66,11 @@ def load_state_dict_to_megatron_qwen2(state_dict,
     def _get_gpt_model(model):
         return model
 
-    def broadcast_params(module):
+    def fetch_params(module):
         for param in module.parameters():
-            torch.distributed.broadcast(param.data,
-                                        src=mpu.get_data_parallel_src_rank(),
-                                        group=mpu.get_data_parallel_group())
+            torch.distributed.fetch(param.data,
```

Comment:
What's this? I cannot find torch.distributed.fetch until pytorch2.6. Do you mean send/recv?